Bottom: 0dbbf1a9ae9d915ec521c6509739bce5b864a66a
Top:    6b6d4815c4cb732d29f22e28a0c7ec28573014f5
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 11:34:47 +0800

Refresh of autofs-5.0.7-depricate-nosymlink-pseudo-option.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 7cf4847..7939daf 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -30,6 +30,7 @@
 - workaround missing GNU versionsort extension.
 - fix wildcard multi map regression.
 - fix file descriptor leak when reloading the daemon.
+- depricate nosymlink pseudo option.
 
 25/07/2012 autofs-5.0.7
 =======================
